pandas/_libs/tslibs/offsets.pyx

```diff
@@ -319,7 +319,8 @@ cdef _determine_offset(kwds):
             # sub-daily offset - use timedelta (tz-aware)
             offset = timedelta(**kwds_no_nanos)
     else:
-        offset = timedelta(1)
+        offset = relativedelta(**{'days':1})
```

Comment:
@mroeschke I don't know how we can easily deprecate this, nor can we simply change this.

Reply:
Can't simply change this in the next release, but I think we could deprecate the keyword arguments and slate for removal in 2.0.